package.json

```diff
@@ -127,7 +127,7 @@
     "@types/uuid": "^8.3.1",
     "@types/wavesurfer.js": "^2.0.2",
     "codelyzer": "^6.0.2",
-    "cypress": "^9.0.0",
+    "cypress": "9.0.0",
```

Comment:
Cypress versions above 9 are failing in containers. 
Upgrading to version 10 requires a change in test structure.